### PR TITLE
Heba Alex-HM1 

### PR DESCRIPTION
**Pull request by HebaAlex**
The process is getting more comfortable, but this platform is complicated.

### CODE REVIEW

**Review by deblnia — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |        X      |            |
| **Coding strategy**         |           |     X         |            |
| **Presentation**            |           |      X        |            |
| **Achievement, creativity** |           |    X          |            |
| **Ease of access**          |           |      X        |            |

Remarks:

*  Minimum requirements met (4 aspects of markdown syntax demonstrated).
* The repository is clean (there are no extraneous files). 
* Reflect more specifically on the assignment and your Github workflow in future pull requests and be sure to commit as often as you'd save an essay.

**Comment by kfurrow** [comment]
## Evaluation

Topic | Excellent | Satisfactory | Needs Work
-- | -- | -- | --
Coding style |  X  |  |  
Coding strategy |   | X |  
Presentation | X  |  |  
Achievement, creativity |   | X |  
Ease of access  |  X | 

Remarks: 
I don't have much to say about your work, other than good job. You accomplished the task set forward well by covering all of the formatting tasks suggested in the homework assignment, and you made it easy to see your file here on GitHub. 

I appreciate that you used a photo from a website; I used one off of my computer, so seeing how to link to something online was appreciated!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
 # hw01
 Repository for homework 01
+## Heba's homework 
+*I hope this line to appear italic*  
+
+I have two difficulties:
+1. The professor is proceeding very fast :) 
+1. The book is hard to follow
```
Comment (kelseypagorek) [inline README.md:8]: I am surprised by your list here. Making a list, you used "1." in the syntax for both points, but the list was formatted correctly. I did not know you could do that. Thank you for teaching me something about Markdown!

**Review by kelseypagorek — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |           |       x       |            |
| **Presentation**            |           |       x       |            |
| **Achievement, creativity** |           |       x       |            |
| **Ease of access**          |      x     |              |            |
I found your assignment to be very clear. Good job!
